src/tools/MappingTools.ts

```diff
@@ -533,6 +535,49 @@ export function mapParameter(
   }
   param.setIdentifier((<ts.Identifier>parameter.name).text);
   if (parameter.type) param.setType(mapTypes(parameter.type));
+  if (parameter.initializer) {
+    switch (parameter.initializer.kind) {
```

Comment:
Would it be possible to refactor this switch as a method? I see that it is pretty similar to the one from [above](https://github.com/devonfw/ts-merger/pull/57/files#diff-a24cb58b71691397454eab7f5fc94127R96). We should try to remove duplicates when possible :)